_pages/pt/developer-engagement.md

```diff
@@ -9,11 +9,11 @@ ref: developer-engagement
 {: .right}
 ![image](/asserts/img/development.png)
 
-Developers are very important for Mozilla. Everything we do is not only to serve our hundreds of millions of users, but to bring Mozilla's manifesto and values to web developers and make their lives easier. We provide many things for developers. To name just a few, we have first class developer tools in Firefox, a world leader add-ons platform, and widely recognised developer documentation portal in MDN.
+Desenvolvedores são muito importantes para Mozilla. Tudo que nós fazemos não é para somente servir nossas centenas de milhares de usuários, mas para trazer o manifesto da Mozilla e valores para desenvolvedores web e fazer suas vidas mais fácil. Nós provemos muitas coisas para desenvolvedores. Para citar apenas algumas, nós temos a primeira ferramenta de desenvolvimento em classe no Firefox, um plataforma de liderança global de add-ons, e o amplamente reconhecido portal de documentação para desenvolvedores o MDN.
```

Comment:
You could translate `add-on` as `complemento`, since we call it that way on Firefox PT-BR.